docs/docs/concepts/dev-environments.md

```diff
@@ -187,10 +187,16 @@ resources:
 
 </div>
 
+The `cpu` property also allows you to specify the CPU architecture, `x86` or `arm`. Examples:
```

Comment:
Need to add `cpu` in the YAML example above too, right?

Reply:
Added